tasks/shell_pcs/cluster-setup-keys.yml

```diff
@@ -34,18 +34,29 @@
     mode: 0400
   register: __ha_cluster_distribute_pacemaker_authkey
 
-- name: Remove qdevice certificates
+- name: Remove qdevice certificates [CLI]
   command:
     cmd: pcs -- qdevice net-client destroy
   when:
     - not ansible_check_mode
     - __ha_cluster_qdevice_model != "net" or ha_cluster_regenerate_keys
+    - ('corosync.quorum.device.client.model.net.certificates.rest-api.v2'
+      not in __ha_cluster_pcsd_capabilities)
   changed_when: true
 
-- name: Obtain and distribute qdevice certificates
+- name: Remove qdevice certificates [API]
+  pcs_api_v2:
+    cmd_name: qdevice.client_net_destroy
+  when:
+    - not ansible_check_mode
+    - __ha_cluster_qdevice_model != "net" or ha_cluster_regenerate_keys
+    - ('corosync.quorum.device.client.model.net.certificates.rest-api.v2'
```

Comment:
Why do you use brackets here? I think using double quotes instead of brackets is conventional in this case.

Reply:
It was my first idea to use brackets. I'll remember using double quotes next time. Thanks.